packages/app/documents/db.ts

```diff
@@ -253,6 +253,22 @@ class DocumentsDb {
         return firstPublicationsEntry['x-meditor'].publishedTo
     }
 
+    /**
+     * removes all document publications for a given Mongo document _id
+     */
+    async removeAllDocumentPublications(documentId: string, modelName: string) {
+        await this.#db.collection(modelName).updateOne(
+            {
+                _id: new ObjectId(documentId),
+            },
+            {
+                $set: {
+                    'x-meditor.publishedTo': {},
```

Comment:
Resets publication statuses to empty, using the mongo _id gives us the exact document version